packages/patternlab/styleguide/source/assets/js/modules/utilNav.js

```diff
@@ -1,9 +1,34 @@
-export default (function (window, document, $, undefined) {
+import focusTrapping from "../helpers/focusTrapping.js";
 
+export default (function (document,$) {
+  const wideContainerClass = '.js-utility-nav--wide .js-util-nav-content';
+  const narrowContainerClass = '.js-utility-nav--narrow .js-util-nav-content';
   let $panels = $('.js-util-nav-content');
   let $panel = "";
   let $utilityButtons = $('.js-util-nav-toggle');
 
+  // Keyboard navigation.
+  $(document).keydown(function(e) {
+    // check if menu open
+    if (!$(wideContainerClass).hasClass('is-closed')) {
+      focusTrapping({
+        focusableSelectors: '.js-utility-nav--wide .js-util-nav-content a, .js-utility-nav--wide .js-util-nav-content button',

```

Comment:
```suggestion
        focusableSelectors: 'a, button',
```
because these selectors are already inside of modalSelector, so no need to add those as parent selectors

Reply:
Not using the class name would cause picking up ones for the utility nav focusable elements in the menu container `.js-utility-nav--narrow`?

Reply:
Ah, got it.  I missed that part.